### ISSUE TITLE
[Feature Request] Separation of Sphere.ini Event definitions

### ISSUE THREAD

**Issue by canerksk** [Sphereserver/Source-X#1141]
m_sEventsItem
m_sEventsPet
m_sEventsPlayer 

In addition to these definitions;

m_sEventsChar <-- All Chars
m_sEventsItem  <-- Only Items
m_sEventsNPC  <-- Only NPC Chars
m_sEventsPlayer  <-- Only Player Chars
m_sEventsStaff <-- Only Staff Chars

Event separations can be made easier by adding

_**Why do we want this?**_

There is only the m_sEventsPlayer definition here, and we have to perform additional operations when we want to give an event for all authorized characters or NPCs.

**Comment by Soulless-1** [comment]
I agree with this thought, with minor suggestions to make it more fine tuned:

m_sEventsItem <-- Only Items
m_sEventsChar <-- All Chars
m_sEventsNPC <-- Only NPC Chars
m_sEventsPet <-- Only Pets
m_sEventsPlayer <-- Only Player Chars
m_sEventsStaff <-- Only Staff Chars
